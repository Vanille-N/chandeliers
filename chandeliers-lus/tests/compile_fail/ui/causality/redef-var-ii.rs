//! Two node parameters with the same name.
//! This would be caught by rustc but we might as well get a nicer error message
//! early because several stages of sanitization would not like this.
chandeliers_lus::decl! {
    node foo(a, a : int) returns ();
    let tel
}

fn main() {}
