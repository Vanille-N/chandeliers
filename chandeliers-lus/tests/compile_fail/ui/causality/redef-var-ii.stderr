error: Attempt to redefine a, when a prior item already defines it
 --> tests/compile_fail/ui/causality/redef-var-ii.rs:5:17
  |
5 |     node foo(a, a : int) returns ();
  |                 ^
  |
note: Already defined here
 --> tests/compile_fail/ui/causality/redef-var-ii.rs:5:14
  |
5 |     node foo(a, a : int) returns ();
  |              ^
